### DISCUSSION TITLE
chia holiday token

### DISCUSSION THREAD

**Discussion by inuxia** [Chia-Network/chia-blockchain#12278] [Support]
Good afternoon, how is everything going? Some time ago you announced that you were going to give away an exclusive token to people who had won a block before you distributed the reward. The case is that I won two blocks and at no time have I received any reward related to that. I am going to attach the address with which I won the two blocks and my main address so that you can see that in no case have I received the said token.
Thanks for all,
Pau Colomé Masferrer.
At your disposal.

Attached:
Main address xch1e2wg4pea2fxfqmr9wp38l5eukrarwfqhpm26ks4zucdsdxzfld9qxnpmna
Earned blocs: xch134nmc5muscw3f203j638y76qps5vqwxz7tmvfvcwjslnhtc0hw7sff8wt0

**Comment by loppefaaret** [comment]
`xch1e2wg4pea2fxfqmr9wp38l5eukrarwfqhpm26ks4zucdsdxzfld9qxnpmna` only has a few transactions on it, in '21 - none of which are "farmer " rewards
`xch134nmc5muscw3f203j638y76qps5vqwxz7tmvfvcwjslnhtc0hw7sff8wt0` only shows "pool" rewards

the latter address shows a pool reward from '21 - looking like it won on https://xchscan.com/blocks/1101443
the farmer address on that block was `xch1gtes2w5m8hf9cd38s2g6fnktgvy9jt29xvy6y9v658g30vpa99cqajqd4j`
and that is showing CH21 tokens here:
https://www.spacescan.io/xch/address/xch1gtes2w5m8hf9cd38s2g6fnktgvy9jt29xvy6y9v658g30vpa99cqajqd4j
